### DISCUSSION TITLE
Error on soup upgrade to 2.3.100 - Rendering SLS 'base:logstash' failed

### DISCUSSION THREAD

**Discussion by TheRealPancakes** [Security-Onion-Solutions/securityonion#7046] [Unsupported Versions] [answered]
Same error occurred on two different standalone sensors. On one, an additional call to 'salt-call state.highstate' seems to resolved the issue. On the other, the error persists.

[soup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7975221/soup.log)
[highstate.log](https://github.com/Security-Onion-Solutions/securityonion/files/7975223/highstate.log)

**Comment by TheRealPancakes** [comment, marked as answer]
Hmmm, sixth time is a charm? I was able to clear up the error on the second sensor by repeatedly retrying 'salt-call state.highstate'.

**Comment by m0duspwnens** [comment]
Can you post the complete soup log?

**Reply by TheRealPancakes** [reply]
Some additional information. It looks like encountering this error also partially breaks the upgrade process. When I load kibana on the affected nodes, the "Security Onion - All Logs" panel is blank ( "No results found"), and I can see a script exception generated when querying a number of shards.
[script_exception.txt](https://github.com/Security-Onion-Solutions/securityonion/files/7980290/script_exception.txt)

**Reply by TOoSmOotH** [reply]
@TheRealPancakes Can you run salt-call mine.get \* network.ip_addrs on the manager and make sure that returns the IP addresses for all your nodes?

**Reply by m0duspwnens** [reply]
@TheRealPancakes For the Kibana issue, can you run the following on your manager.

`salt-call state.apply kibana.so_savedobjects_defaults -l info`

This state will load all the default saved objects for Kibana.

**Reply by TheRealPancakes** [reply]
> @TheRealPancakes Can you run salt-call mine.get * network.ip_addrs on the manager and make sure that returns the IP addresses for all your nodes?

If I run `salt '*' mine.get '*' network.ip_addrs` I do see the IP address (these are standalone installations) :

```
REDACTED_standalone:
    ----------
    REDACTED_standalone:
        - 10.WWW.XXX.YYY
```

If I run `salt-call mine.get '*' network.ip_addrs` I get this :

```
local:
    ----------
    REDACTED_standalone:
        - 10.WWW.XXX.YYY
```

There are actual 10./8 IP addresses displayed, I've just redacted them to 10.WWW.XXX.YYY

**Reply by TheRealPancakes** [reply]
> @TheRealPancakes For the Kibana issue, can you run the following on your manager.
> 
> `salt-call state.apply kibana.so_savedobjects_defaults -l info`
> 
> This state will load all the default saved objects for Kibana.

This task completed successfully and repaired the kibana dashboard. Thanks!

**Comment by 0nnyx** [comment]
Got exactly the same traceback as in your soup.log when upgrading from 2.3.91 to 2.3.100 Standalone. Didn't have to apply highstate and SO seems to be working properly.

**Reply by m0duspwnens** [reply]
Can you post your complete soup.log?

**Reply by 0nnyx** [reply]
Had to reboot SO due to OS updates. soup.log is apparently stored in /tmp hence erased after reboot.
Seems we've hit this scenario : https://docs.securityonion.net/en/2.3/soup.html#highstate-already-running

**Reply by TOoSmOotH** [reply]
The soup log is in /root

**Reply by 0nnyx** [reply]
Indeed it's in /root but won't upload it as it's 1.5gig. Might be an idea to clean this log periodically

**Reply by TOoSmOotH** [reply]
That file should not be that large. Are you running soup via a cron job or something?

**Reply by 0nnyx** [reply]
I only run soup when an update is published but my initial version was 2.3.10 or 2.3.21

**Reply by TOoSmOotH** [reply]
@stondino00 Start a new thread please

**Reply by 0nnyx** [reply]
Noticed indeed couple of things broken after the upgrade:
- **script exception** in Kibana : Fixed with _salt-call state.apply kibana.so_savedobjects_defaults -l info_ as reported at https://github.com/Security-Onion-Solutions/securityonion/discussions/7056

- **Request failed with status code 400** when trying to Escalate alerts to a case or creating a native case (not Hive case)
